test/utils.py

```diff
@@ -337,17 +337,17 @@ async def asleep_coroutine(self, duration=3, times=1, sequentially=False, **kwar
             for _ in range(times):
                 await self.asql(f"select pg_sleep({duration})", **kwargs)
 
-    def test(self, **kwargs):
+    def test(self, query="select 1", **kwargs):
```

Comment:
Instead of allowing to pass a query to `test` I think it's better to use the `sql` method directly in the places where you want to use a different query. The intent of the `test` functions is really just to test if the connection is not broken.

Reply:
Thank you for pointing it out.
I have just reverted the changes in utils.py in the `QueryRunner` class and implemented tests using `bouncer.sql`